src/odemis/gui/cont/multi_point_correlation.py

```
@@ -705,102 +687,95 @@ def _on_current_coordinates_changes(self, coordinates: ListVA) -> None:
+
             coords = self._tab_data_model.main.currentTarget.value.coordinates.value
-            pixel_coords = self.correlation_target.fm_streams[0].getPixelCoordinates((coords[0], coords[1]),
-                                                                                     check_bbox=False)
-            self._tab_data_model.main.currentTarget.value.coordinates.value[2] = float(get_optimized_z_gauss(das, int(
-                pixel_coords[0]), int(pixel_coords[1]), coords[2]))
+            pixel_coords = self.correlation_target.fm_streams[0].getPixel3DCoordinates(coords)
+            das = [interpolate_z_stack(da=stream_projection.stream.raw[0][:,int(pixel_coords[1]):int(pixel_coords[1])+1,
+                   int(pixel_coords[0]):int(pixel_coords[0])+1], method="linear")
+                   for stream_projection in streams_projections]
+
+            z = float(get_optimized_z_gauss(das,int(0),int(0),coords[2]))
```

Ruff (0.14.0)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)